src/utils/time_utils.cpp

```diff
@@ -38,37 +60,54 @@ std::string time_utils::timepoint_to_readable(const time_utils::t_timepoint &tp,
 
 std::string time_utils::time_t_to_readable(const std::time_t &time, const std::string &zone) {
 
+	// Pacific/Marquesas and America/St_Johns warn
+	// This two zones might not works properly.
+	// However it looks like boost::date::time fixed this
+	if(zone == "Pacific/Marquesas" || zone == "America/St_Johns") {
```

Comment:
It looks like boost::date_time fixed this, but I am still not sure so I decided to leave warn.